apps/condo/domains/organization/components/DeleteEmployeeButtonWithReassignmentModel.tsx

```diff
@@ -0,0 +1,248 @@
+
+import { GraphQlSearchInput } from '@condo/domains/common/components/GraphQlSearchInput'
+import { runMutation } from '@condo/domains/common/utils/mutations.utils'
+import { sleep } from '@condo/domains/common/utils/sleep'
+import { searchEmployeeUser } from '@condo/domains/ticket/utils/clientSchema/search'
+
+
+type IDeleteEmployeeButtonWithReassignmentModel = {
+    buttonContent: string
+    softDeleteAction: IUseSoftDeleteActionType<OrganizationEmployee>
+    disabled?: boolean
+    employee: Pick<OrganizationEmployee, 'name'> & { organization?: 
+    Pick<OrganizationEmployee['organization'], 'id'>, user?: 
+    Pick<OrganizationEmployee['organization'], 'id'> }
+    activeTicketsOrganizationEmployeeCount: number
+}
+
+const ERROR_NOTIFICATION_TYPE = 'error'
+const WARNING_NOTIFICATION_TYPE = 'warning'
+const SUCCESS_NOTIFICATION_TYPE = 'success'
+
+const waitBetweenRequsted = async () => await sleep(1000)
+
+export const DeleteEmployeeButtonWithReassignmentModel: React.FC<IDeleteEmployeeButtonWithReassignmentModel> = ({
+    buttonContent,
+    softDeleteAction,
+    disabled = false,
+    employee,
+    activeTicketsOrganizationEmployeeCount,
+}) => {
+    const intl = useIntl()
+    const ConfirmReassignEmployeeTitle = intl.formatMessage({ id: 'employee.reassignTickets.title' })
+    const ConfirmDeleteButtonLabel = intl.formatMessage({ id: 'employee.reassignTickets.buttonContent.deleteUser' })
+    const ConfirmReassignTicketsButtonLabel = intl.formatMessage({ id: 'employee.reassignTickets.buttonContent.reassignTickets' })
+    const SearchPlaceholderLabel = intl.formatMessage({ id: 'EmployeesName' })
+    const AlertTitleLabel = intl.formatMessage({ id: 'employee.reassignTickets.alert.title' }, { employeeName: employee?.name || null })
+    const CountShortLabel = intl.formatMessage({ id: 'global.count.pieces.short' })
+    const AlertMessageLabel = intl.formatMessage({ id: 'employee.reassignTickets.alert.message' })
+    const NotificationTitleWarningLabel = intl.formatMessage({ id: 'employee.reassignTickets.notification.title.warning' })
+    const NotificationTitleErrorLabel = intl.formatMessage({ id: 'employee.reassignTickets.notification.title.error' })
+    const NotificationTitleSuccessLabel = intl.formatMessage({ id: 'employee.reassignTickets.notification.title.success' })
+    const NotificationMessageWarningLabel = intl.formatMessage({ id: 'employee.reassignTickets.notification.message.warning' })
+    const NotificationMessageErrorLabel = intl.formatMessage({ id: 'employee.reassignTickets.notification.message.error' })
+    const NotificationMessageSuccessLabel = intl.formatMessage({ id: 'employee.reassignTickets.notification.message.success' })
+
+    const [notificationApi, contextHolder] = notification.useNotification()
+
+    const employeeUserId = employee?.user?.id || null
+    const employeeOrganizationId = employee?.organization?.id || null
+
+    const getTicketReassignData = (ticket: Ticket) => {
```

Comment:
This parameter is not of type `Ticket`. Use the correct generated type for your request (from `@app/condo/gql`)